docker/nginx.dockerfile

```
@@ -0,0 +1,11 @@
+from nginx:latest
```

```suggestion
FROM nginx:latest
```
I'm just anal about consistency sorry :sweat_smile: